### ISSUE TITLE
IDEA 2021.2: ClassNotFoundException: CheckstyleActionsImpl

### ISSUE THREAD

**Issue by calohmn** [jshiell/checkstyle-idea#530]
I'm using Checkstyle IDEA 5.54.0 and have just updated to IDEA 2021.2.

Since the update, scans fail with:
```
Unexpected Exception Caught
The scan failed due to an exception: internal error
Root cause:
org.infernus.idea.checkstyle.service.CheckstyleActionsImpl
java.lang.ClassNotFoundException: org.infernus.idea.checkstyle.service.CheckstyleActionsImpl
at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:471)
at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:589)
at org.infernus.idea.checkstyle.util.ChildFirstURLClassLoader.loadClass(ChildFirstURLClassLoader.java:41)
at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
at org.infernus.idea.checkstyle.CheckstyleClassLoaderContainer.loadCheckstyleImpl(CheckstyleClassLoaderContainer.java:243)
at org.infernus.idea.checkstyle.CheckstyleProjectService.getCheckstyleInstance(CheckstyleProjectService.java:111)
at org.infernus.idea.checkstyle.checker.CheckerFactoryWorker.run(CheckerFactoryWorker.java:45)
```

**Comment by jshiell** [comment]
I fear I can't reproduce this on 2021.2 - everything seems to be working nicely here.

In the past this sort of thing has been caused by IDEA buggering up the plugin files locally - could you please try removing the plugin entirely and reinstalling it?

**Comment by calohmn** [comment]
Thanks, removing and reinstalling the plugin has indeed resolved the issue.
Maybe it was caused by the broken (and then removed) 2021.2 update mechanism. I had tried and then cancelled the update and later reinstalled IDEA. _Maybe_ the `.local/share/JetBrains/IntelliJIdea2021.1` directory was already copied during the update and was causing the issue (although I didn't see problems with other plugins).
Closing the issue here.
